src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -7069,7 +7060,8 @@ void WriteMonthlyTables(EnergyPlusData &state)
                         curAggString = " {" + stripped(curAggString) + '}';
                     }
                     // do the unit conversions
-                    if (unitsStyle_cur == UnitsStyle::InchPound) {
+                    if (unitsStyle_cur == OutputReportTabular::UnitsStyle::InchPound ||
+                        unitsStyle_cur == OutputReportTabular::UnitsStyle::InchPoundExceptElectricity) {
```

Comment:
Keep in mind though, that the ip() function only checks the value of unitsStyle defined currently on the ORT state instance.  So if you are comparing some other style, that function isn't the right one to use, so I just left these remaining ones as `if(thisUnitsStyle == A || thisUnitsStlye == B)`